src/behaviours/add-cluster/navigation-using-application-menu.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { RenderResult } from "@testing-library/react";
+import { ApplicationBuilder, getApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import isAutoUpdateEnabledInjectable from "../../main/is-auto-update-enabled.injectable";
+
+// TODO: Make components free of side effects by making them deterministic
+jest.mock("../../renderer/components/tooltip");
+jest.mock("../../renderer/components/monaco-editor/monaco-editor");
+
+describe("add-cluster - navigation using application menu", () => {
+  let applicationBuilder: ApplicationBuilder;
+  let rendered: RenderResult;
+
+  beforeEach(async () => {
+    applicationBuilder = getApplicationBuilder().beforeSetups(({ mainDi }) => {
+      mainDi.override(isAutoUpdateEnabledInjectable, () => () => false);
+    });
+
+    rendered = await applicationBuilder.render();
+  });
+
+  it("renders", () => {
+    expect(rendered.container).toMatchSnapshot();
+  });
+
+  it("does not show welcome page yet", () => {
+    const actual = rendered.queryByTestId("add-cluster-page");
+
+    expect(actual).toBeNull();
+  });
+
+  describe("when navigating to add cluster using application menu", () => {
```

Comment:
First snapshot test in this suite is snapshot that will match to "front page" of the application. However, currently it's just component that doesn't render any page component. Second snapshot matches to "add-cluster-page". 

- `it("shows add cluster page", () => {` tests that we are actually on that page and   
- `it("renders", () => {` tests how it's rendered at the moment. Covers a lot, since all of the sub components are rendered too and if there is change, it will break the snapshot and make sure you are attempting to change functionality in this page.

This is just a behaviour for navigating to that page. In future, there will be a lot more behaviours for "add-cluster-page" which will cover all of the functionalities. Most of these will be covered in other behaviours that actually do some action.

Examples of behaviours:
- Add single cluster
- Add multiple clusters
- Cancel 
- Navigate using application menu
- Navigate from catalog (most likely covered with all other behaviours)